src/engine/strat_engine/names.rs

```diff
@@ -12,12 +12,21 @@ use std::{
 use devicemapper::{DmNameBuf, DmUuidBuf};
 
 use crate::{
-    engine::types::{FilesystemUuid, PoolUuid},
+    engine::types::{DevUuid, FilesystemUuid, PoolUuid},
     stratis::{ErrorEnum, StratisError, StratisResult},
 };
 
 const FORMAT_VERSION: u16 = 1;
 
+/// Get a devicemapper name from the device UUIDs.
```

Comment:
Please change to singular.

Also, put in proof the name is short enough as:

```
Prerequisite: len(format!("{}", FORMAT_VERSION) 
                     + len("stratis")         7
                     + len("private")        7
                     + len("crypt")            5
                     + num_dashes        4
                     + len(dev_uuid)     32
                     < 128
which is equivalent to len(format!("{}", FORMAT_VERSION) < 73
```

see examples below.

Reply:
Resolved.